lib/widgets/fullscreen_loader.dart

```diff
@@ -46,52 +59,48 @@ class _FullscreenLoaderState extends State<FullscreenLoader>
     super.dispose();
     animationController.dispose();
 
-    if (statusSubscription != null)
-      statusSubscription.cancel();
+    if (statusSubscription != null) statusSubscription.cancel();
 
-    if (messageSubscription != null)
-      messageSubscription.cancel();
+    if (messageSubscription != null) messageSubscription.cancel();
   }
 
   @override
   void initState() {
-    print("init loader");
-
     super.initState();
 
     if (widget.messageStream != null) {
       messageSubscription = widget.messageStream.listen((resultMessage) {
-        print("new message");
         setState(() {
           message = resultMessage;
         });
       });
     }
 
+    print("listen now...");
```

Comment:
print statements should be removed or commented out, unless they're important (error conditions, mostly)

Reply:
Ok